internal/common/mqtt/mqttconnection.go

```diff
@@ -150,3 +150,25 @@ func (client *Client) Publish(message Message, topic string) error {
 	time.Sleep(time.Second)
 	return nil
 }
+
+var messageHandler MQTT.MessageHandler = func(client MQTT.Client, msg MQTT.Message) {
+
+	log.Printf("Topic %s registered.. %s is the payload \n", msg.Topic(), msg.Payload())
+}
+
+//Subscribe is used to subscribe to a topic
+func (client *Client) Subscribe(topic string) error {
+
+	log.Info(logPrefix, "Subscribing the data to cloud")
```

Comment:
Please remove empty line and it would be great to print with topic information.
```suggestion
	log.Info(logPrefix, "Subscribing topic: ", topic)
```

Reply:
Done!